.ci/setup_env_ubuntu.sh

```diff
@@ -112,3 +112,15 @@ make nsenter
 sudo cp nsenter /usr/bin/
 popd
 rm -rf util-linux-${nsenter_version}/ util-linux-${nsenter_version}.tar.xz
+
+echo "Install ostree"
+ostree_dir="ostree"
+chronic sudo -E apt-get install -y liblzma-dev e2p-dev libfuse-dev gtk-doc-tools
+git clone https://github.com/ostreedev/ostree.git
```

Comment:
just a passing note - you might be able to do a 'shallow clone' here, as we will only be building from the tip (HEAD) of the source, and thus save us some bandwidth and time.